docs/resources/Guild_Scheduled_Event.md

```diff
@@ -6,54 +6,99 @@ A representation of a scheduled event in a [guild](#DOCS_RESOURCES_GUILD/).
+| description?          | string                                                                                                                         | the description of the scheduled event (1-1000 characters)                                          |
+| scheduled_start_time  | ISO8601 timestamp                                                                                                              | the time the scheduled event will start                                                             |
+| scheduled_end_time ** | ?ISO8601 timestamp                                                                                                             | the time the scheduled event will end, required if entity_type is `EXTERNAL`                        |
+| privacy_level         | [privacy level](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-guild-scheduled-event-privacy-level)        | the privacy level of the scheduled event                                                            |
+| status                | [event status](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-guild-scheduled-event-status)                | the status of the scheduled event                                                                   |
+| entity_type           | [scheduled entity type](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-guild-scheduled-event-entity-types) | the type of the scheduled event                                                                     |
+| entity_id             | ?snowflake                                                                                                                     | the id of an entity associated with a guild scheduled event                                         |
+| entity_metadata **    | ?[entity metadata](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-guild-scheduled-event-entity-metadata)   | additional metadata for the guild scheduled event                                                   |
+| creator?              | [user](#DOCS_RESOURCES_USER/user-object) object                                                                                | the user that created the scheduled event                                                           |
+| user_count?           | integer                                                                                                                        | the number of users subscribed to the scheduled event                                               |
+
+\* `creator_id` will be null and `creator` will not be included for events created before October 25th, 2021, when the concept of `creator_id` was introduced and tracked.
```

Comment:
```suggestion
> note
> `creator_id` will be null and `creator` will not be included for events created before October 25th, 2021, when the concept of `creator_id` was introduced and tracked.
```

Making this a note will be better for consistency of the docs, and illustrates importance for data model purposes. (Plus I think it looks better)